vacancies-aggregator/src/main/resources/application.properties

```diff
@@ -0,0 +1,5 @@
+spring.datasource.url=jdbc:postgresql://localhost:5432/vacancies-aggregator
+spring.application.name=vacancies-aggregator
+spring.datasource.username=postgres
+spring.datasource.password=postgres
```

Comment:
don't use parameters database for development, for it need to install plugin for IDE EnvFile